route_manager:
  ros__parameters:
    start_label: ""
    goal_label: ""
    checkpoint_labels: ["C4","C6","C13"]
    planner_timeout_sec: 5.0
    planner_retry_count: 2
    planner_connect_timeout_sec: 10.0
    state_publish_rate_hz: 1.0
    image_encoding_check: false
    report_stuck_timeout_sec: 5.0
    offset_step_max_m: 1.0
